config-windows.md

```diff
@@ -77,11 +77,13 @@ The following parameters can be specified:
 
 `cpu` is an OPTIONAL configuration for the container's CPU usage.
 
-The following parameters can be specified:
+The following parameters can be specified (mutually exclusive):
+
+* **`count`** *(uint64, OPTIONAL)* - specifies the number of CPUs available to the container. It represents the fraction of the configured processor count in a container in relation to the processors available in the host. The fraction ultimately determines the portion of processor cycles that the threads in a container can use during each scheduling interval, as the number of cycles per 10,000 cycles.
+* **`shares`** *(uint16, OPTIONAL)* - limits the share of processor time given to the container relative to other workloads on the processor. The processor weight is a value between 0 and 10,000.
```

Comment:
nit: The OCI naming is shares but the last sentence is worded as if Weight, like on the HCS schema, is the field name here also. Maybe note a "also referred to as processor weight" or change the end to "Shares is a value between 0 and 10,000" or something similar.

Reply:
Not sure why this selected 3 extra lines.. This was directed at line 83 (shares) only

Reply:
Lol I get what you're saying. Will do an update. I've always found it confusing for OCI we used Shares even though the platform does not have it. Will update

Reply:
Ok done.

Reply:
Thanks! Resolving

Reply:
Oh never mind I can't, feel free to resolve 😆